Makefile

```diff
@@ -0,0 +1,11 @@
+# It seems that the venv doesn't work in the makefile. https://stackoverflow.com/questions/44052093/makefile-with-source-get-error-no-such-file-or-directory
+# If you want to use virtual environment, you can create the virtual environment first, then run the make file, here is the example
+# python3 -m venv .venv
+# source .venv/bin/activate
+
+sdk_dev_install:
+	pip install -e sdk/python
```

Comment:
we should not do a `pip install` outside a virtual environment

Reply:
I made a make help, the user can choose to do the install inside the `venv`.